### ISSUE TITLE
Broken graph.dot file for some bridge names

### ISSUE THREAD

**Issue by ssasso** [ipspace/netsim-tools#244]
In some cases, the bridge name breaks the *graph.dot* file:

https://github.com/ipspace/netsim-tools/blob/d1b137c0fc4097675c1a0b863598061f65cf67f7/netsim/augment/links.py#L563

Example:
```
graph {
  bgcolor="transparent"
  node [shape=box, style="rounded,filled" fontname=Verdana]
  edge [fontname=Verdana labelfontsize=10 labeldistance=1.5]
  subgraph cluster_65000 {
    bgcolor="#e8e8e8"
    fontname=Verdana
    margin=16
    label="AS 65000"
    core1 [
      label=<core1 [dellos10]<br /><sub>10.0.0.1/32</sub>>
      fillcolor="#ff9f01"
    ]
  }
  subgraph cluster_65111 {
    bgcolor="#e8e8e8"
    fontname=Verdana
    margin=16
    label="AS 65111"
    aggr2 [
      label=<aggr2 [dellos10]<br /><sub>10.0.0.3/32</sub>>
      fillcolor="#ff9f01"
    ]
  }
 aggr1 -- core1 [ ]
 aggr2 -- core1 [ ]
  netsim-br_3 [ style=filled fillcolor="#d1bfab" fontsize=11 label="172.16.0.0/24" ]
 aggr1 -- netsim-br_3 [ ]
 host1 -- netsim-br_3 [ ]
  netsim-br_4 [ style=filled fillcolor="#d1bfab" fontsize=11 label="172.16.1.0/24" ]
 aggr2 -- netsim-br_4 [ ]
 host2 -- netsim-br_4 [ ]
  netsim-br_5 [ style=filled fillcolor="#d1bfab" fontsize=11 label="172.16.2.0/24" ]
 core1 -- netsim-br_5 [ ]
 server -- netsim-br_5 [ ]
}
```
will cause:
```
Error: graph.dot: syntax error in line 27 near '-'
```

It **seems** the issue can be resolved using `_` instead of `-` on the bridge name.
If you don't see any issue with that, I can submit a patch.

**Comment by ssasso** [comment]
The problem if present also if `defaults.name` contains `-`.

I think we have two options:

1. search/replace any `-` with `_` when setting the bridge name (on *augment/links.py*)
2. search/replace any `-` with `_` on the graph generation module only

**Comment by ipspace** [comment]
Found the third option: quote node names in graph.dot (implemented in 4e652fd) -- who knows what other special characters might make graphviz unhappy.
